src/writer/fail_on_skipped.rs

```diff
@@ -118,6 +118,7 @@ where
                 ) => map_failed_step(f, None, sc, st),
                 Cucumber::Started
                 | Cucumber::Feature(..)
+                | Cucumber::ParsingFinished { .. }
```

Comment:
@ilslv and this event removing is also not obvious. Would be nice if you describe architecture changes more detailed in PR description.

Reply:
@tyranron you mean why this new event is necessary?

First line of the `libtest`'s `json` output is this following:

```
{"type":"suite","event":"started","test_count":41}
```

So we need number of test cases, which is equal to number of parsed steps + number of parsing failures. And because of the streaming nature, the only way to get number of steps before the end of the execution is the additional event.

Reply:
@ilslv yup, now it's clear, thanks. Still, would love to see this in PR description too.